msteams-platform/bots/how-to/authentication/add-authentication.md

```diff
@@ -73,17 +73,79 @@ You use a resource group to create individual resources for the Bot Framework. F
     1. Select the **Review and create** button. You should see a banner that reads *Validation passed*.
     1. Select **Create**. It may take a few minutes to create the app service plan. The plan will be listed in the resource group.
 
-## Create the bot channels registration
+## Create Azure Bot resource registration
 
-The bot channels registration registers your web service as a bot with the Bot Framework, provided you have a Microsoft App Id and App password (client secret).
+The Azure Bot resource registration registers your web service as a bot with the Bot Framework which provides you a Microsoft App Id and App password (client secret).
 
 > [!IMPORTANT]
 > You only need to register your bot if it is not hosted in Azure. If you [created a bot](/azure/bot-service/abs-quickstart?view=azure-bot-service-4.0&viewFallbackFrom=azure-bot-service-3.0&preserve-view=true) through the Azure portal then it is already registered with the service. If you created your bot through the [Bot Framework](https://dev.botframework.com/bots/new) or [Developer Portal](../../../concepts/build-and-test/teams-developer-portal.md) your bot isn't registered in Azure.
 
-[!INCLUDE [bot channels registration steps](~/includes/bots/azure-bot-channels-registration.md)]
+1. Select **Create**.
```

Comment:
Could you please include more details on which resource to create? 

Can we change it to following?
1. Visit Azure portal and search for **Azure Bot** in create new resource section.

Reply:
Changed it accordingly.